common/utils/filters.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  CatalogueAggregationBucket,
+  CatalogueAggregations,
+} from '@weco/common/model/catalogue';
+import { nodeListValueToArray, inputValue } from '@weco/common/utils/forms';
+import { LinkProps } from 'next/link';
+
+import {
+  worksLink,
+  imagesLink,
+} from '@weco/common/services/catalogue/ts_routes';
+
+export const getFilterByCount = (
+  bucketFilter: CatalogueAggregationBucket[]
+): CatalogueAggregationBucket[] => {
+  return bucketFilter.filter(items => {
+    return items.count > 0;
+  });
+};
+
+export const getAggregationFilterByName = (
+  aggregations: CatalogueAggregations | undefined,
+  name: string
```

Comment:
We would type this to the keys of the `CatalogueAggregations` to make sure we always get something back.